src/listr/tasks/platform/initTaskFactory.js

```diff
@@ -55,13 +55,15 @@ function initTaskFactory(
             ...clientOpts,
             wallet: {
               privateKey: ctx.fundingPrivateKeyString,
+              waitForInstantLockTimeout: ctx.nodeCount * 60000,
```

Comment:
We use `initTask` in different commands. I guess `nodeCount` is present only in case of `setupLocalPresetTask`.